### PR TITLE
V2 Main Deploy

### PR DESCRIPTION
**Pull request by Joeysantoro**
Deploys TRIBE Minter and Backstop

### CODE REVIEW

**Comment by ditchfieldcaleb** [comment]
hype af, lfggg

File: proposals/dao/backstop.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { ethers } from 'hardhat';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import {
+  DeployUpgradeFunc,
+  NamedContracts,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '../../types/types';
```
Comment (ditchfieldcaleb) [inline proposals/dao/backstop.ts:10]: use relative path

Comment (Joeysantoro) [reply proposals/dao/backstop.ts]: done

**Review by eswak — APPROVED** [review]

**Comment by ElliotFriedman** [comment]
LGTM!

**Review by ditchfieldcaleb — CHANGES_REQUESTED** [review]
Will approve once e2e test is fixed & merge conflict is fixed.

**Review by ditchfieldcaleb — APPROVED** [review]

**Pull request merged**